### ISSUE TITLE
Nitpicks

### ISSUE THREAD

**Issue by glatard** [poldrack/narps#32]
Probably not worth fixing but still reporting just in case.

1. Not sure if it's really necessary to use global variables here:
https://github.com/poldrack/narps/blob/c6040140573002c4d934046836c2ac1073f51dcc/ImageAnalyses/narps.py#L45-L86

2. The following for loops could be replaced by dict comprehension:
https://github.com/poldrack/narps/blob/c6040140573002c4d934046836c2ac1073f51dcc/ImageAnalyses/narps.py#L114-L115
https://github.com/poldrack/narps/blob/c6040140573002c4d934046836c2ac1073f51dcc/ImageAnalyses/narps.py#L130-L131

3. This could be replaced by a test on `d` in the subsequent for loop:
https://github.com/poldrack/narps/blob/c6040140573002c4d934046836c2ac1073f51dcc/ImageAnalyses/narps.py#L119

4. Here, `self.images['thresh']['thresh_mask_orig'][hyp]` could be put in a variable as it's reused a few times after:
https://github.com/poldrack/narps/blob/c6040140573002c4d934046836c2ac1073f51dcc/ImageAnalyses/narps.py#L262

5. I think this reads the data while you should be able to get the shape without doing that:
https://github.com/poldrack/narps/blob/c6040140573002c4d934046836c2ac1073f51dcc/ImageAnalyses/narps.py#L407

6. For portability, `\n` chars should be replaced by `os.linesep` everywhere.

7. Help text of `-b` option could be updated to:

https://github.com/glatard/narps/blob/5628cef4bcd5694475720456ec48c0741c21e3de/ImageAnalyses/narps.py#L1205-L1207

**Comment by glatard** [comment]
8. The following functions of AnalyzeMaps.py have a lot in common and code could likely be mutualized:
```
    mk_overlap_maps(narps)
    mk_range_maps(narps)
    mk_std_maps(narps)
```

**Comment by glatard** [comment]
9. mean_corr could be computed outside of the for loop instead of using `locals()`:

https://github.com/poldrack/narps/blob/c6040140573002c4d934046836c2ac1073f51dcc/ImageAnalyses/AnalyzeMaps.py#L272

**Comment by poldrack** [comment]
#5 addressed in ae10cf7

**Comment by poldrack** [comment]
#6 addressed in57e5d55

**Comment by poldrack** [comment]
#1 - I am using globals because I want to easily import those in the other scripts. perhaps there is a better way to do it...

**Comment by poldrack** [comment]
#3 - can you clarify what you are suggesting?

**Comment by poldrack** [comment]
#4 fixed in df357ec

**Comment by glatard** [comment]
#1: couldn't they just be class attributes of Narps? 
#3: keep `dir_to_make` intact and then:
```
for d in dirs_to_make:
    if d != 'orig' and not os.path.exists(self.dirs[d]):
    ...
```

**Comment by poldrack** [comment]
#1 - yes, I could do that, though then I would need to actually instantiate the class, whereas now I can just import them.  
#3 - fixed in c6a720a

**Comment by poldrack** [comment]
#7 fixed in c3230df
